### ISSUE TITLE
Please Read.

### ISSUE THREAD

**Issue by ImReallyShiny** [J-D-K/JKSM#28]
Since you blocked me basically everywhere - I need to contact you here,

Feel free to use the code/graphics etc that I modified in JKSMK into your main application.
I dont think ill be updating it much anymore since no one cared about it as they tought I was ripping you off when you clearly said it was OK in pm's.

Anyway feel free to merge the new code and UI etc. One bug I noticed is that the help text doesnt seem to show for whatever reason.

Also a small request (Since you know a lot more on CPP then me), Try implement FBI's menu's into JKSM since it would be a HUGE boost in speed.

**Comment by J-D-K** [comment]
Is there something specific from FBI's menus that makes them faster or easier to use for you? I'm looking to actually remove the text menus almost altogether and have it boot straight to the title select, which will be a lot different too. It's just taking a while because of work.

**Comment by ImReallyShiny** [comment]
Yes, FBI's menus you can hold down/up and it would keep moving and such.
And yeah if I were you keep the initial mainmenu doe so you can add a Extras menu for playcoins etc.
I asked Steven (Steveice) if he plans on adding import/export features and he simply replied saying go into browse save data and click on current directory copy and paste etc which works but the amount of bother doing that...

**Comment by ImReallyShiny** [comment]
Also - If you plan on renaming it to something more linear I had plans on making a FBI branch where it had the import/export but couldnt get it working (Not great at C, Only CPP and C#). I used the name "SaveNav" for it. I have logos and such for it if you need em so hmu. And also a 3d banner :P
